tests/integration/scheduler/test_postpone.py

```diff
@@ -43,10 +43,13 @@ def setUp(self):
             functions = {'test.ping': ping}
             self.schedule = salt.utils.schedule.Schedule(copy.deepcopy(DEFAULT_CONFIG), functions, returners={})
         self.schedule.opts['loop_interval'] = 1
+        self.schedule.opts['run_schedule_jobs_in_background'] = False
 
     def tearDown(self):
         self.schedule.reset()
 
+        del self.schedule
```

Comment:
Why is this needed? If we need some kind of shutdown, can we call the shutdown method directly, or could we just set self.schedule to None and let it get garbage collected?

Reply:
The `reset` function that is being called does set the schedule to an empty dict.  I added the del because the test suit kept recommending it in a log.  I'll take it out.

Reply:
Oh, right. I forgot the test suite will do that. I'm fine with leaving it in that case

Reply:
I took it out and having it as an empty dict is fine, it will be recreated in between each test run.